paasta_tools/kubernetes_tools.py

```diff
@@ -649,13 +650,22 @@ def get_kubernetes_services_running_here_for_nerve(
     return nerve_list
 
 
-class KubeClient():
+class KubeClient(object):
     def __init__(self) -> None:
         kube_config.load_kube_config(config_file='/etc/kubernetes/admin.conf')
         self.deployments = kube_client.AppsV1Api()
         self.core = kube_client.CoreV1Api()
 
 
+class KubeClients(object):
+    def get_current_client_for_service(self, job_config: KubernetesDeploymentConfig) -> KubeClient:
```

Comment:
I think it's better to skip doing this. For Marathon we have logic to get multiple clients because we shard by service instance across multiple marathons. Hopefully we won't be doing the same with k8s anytime soon...